### PR TITLE
travis-ci: add qemu-ax25 (riscv64)

### PR DESCRIPTION
**Pull request by PkmX**
Hi,

This patch is for running u-boot's travis CI tests on our ax25 board port. Thanks.

Signed-off-by: Rick Chen <rick@andestech.com>
Signed-off-by: Rick Chen <rickchen36@gmail.com>
Reviewed-by: Chih-Mao Chen <cmchen@andestech.com>
Cc: Greentime Hu <green.hu@gmail.com>

### CODE REVIEW

File: bin/travis-ci/conf.qemu-ax25_na [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+# DEALINGS IN THE SOFTWARE.
+
+console_impl=qemu
+qemu_machine="andes_ae350"
+qemu_binary="qemu-system-riscv64"
+qemu_extra_args="-nographic -m 4G -gdb tcp::5678"
+qemu_kernel_args="-kernel ${U_BOOT_BUILD_DIR}/u-boot"
```
Comment (swarren) [inline bin/travis-ci/conf.qemu-ax25_na]: Not sure why some config files use -kernel here, whereas others use -bios. Still, it works so no real objection, just a query whether it's optimal?

File: bin/travis-ci/conf.qemu-ax25_na [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+# DEALINGS IN THE SOFTWARE.
+
+console_impl=qemu
+qemu_machine="andes_ae350"
+qemu_binary="qemu-system-riscv64"
+qemu_extra_args="-nographic -m 4G -gdb tcp::5678"
```
Comment (swarren) [inline bin/travis-ci/conf.qemu-ax25_na]: Do you really want/need gdb enabled for a Travis run?

File: bin/travis-ci/conf.qemu-ax25_na (hunk 1 above)
Comment (PkmX) [reply bin/travis-ci/conf.qemu-ax25_na]: Yes, our qemu port can use `-kernel` to load u-boot as an ELF executable directly.

File: bin/travis-ci/conf.qemu-ax25_na (hunk 2 above)
Comment (PkmX) [reply bin/travis-ci/conf.qemu-ax25_na]: No, we don't really need `gdb` and I will update this PR with this option removed.

**Comment by swarren** [comment]
Hmm. Looks like I didn't notice the updated version you pushed. Can you please update the patch so that it doesn't conflict and then I'll take another look? Thanks.

**Comment by swarren** [comment]
This pull request just duplicates files that already exist. Closing.

**Pull request closed without merging**